interfaces/builtin/fuse_support.go

```diff
@@ -68,6 +69,10 @@ mount fstype=fuse.* options=(rw,nosuid,nodev) ** -> /home/*/snap/@{SNAP_NAME}/@{
 mount fstype=fuse.* options=(ro,nosuid,nodev) ** -> /var/snap/@{SNAP_NAME}/@{SNAP_REVISION}/{,**/},
 mount fstype=fuse.* options=(rw,nosuid,nodev) ** -> /var/snap/@{SNAP_NAME}/@{SNAP_REVISION}/{,**/},
 
+# Allow unmounting
+umount fstype=fuse.* /home/*/snap/@{SNAP_NAME}/@{SNAP_REVISION}/{,**/},
+umount fstype=fuse.* /var/snap/@{SNAP_NAME}/@{SNAP_REVISION}/{,**/},
```

Comment:
NAK - see https://github.com/snapcore/snapd/pull/1598#issuecomment-239952977 for details. Specifically, this rule isn't mediating the way you think it is due to https://bugs.launchpad.net/apparmor/+bug/1613403. IIRC, this is why we used `kill -9` previously.

Reply:
As discussed on IRC, I'll drop this particular patch and update the tests to verify that umount is blocked by the confinement policy.

Reply:
Had to force push to drop this patch. Hope you don't mind.